lib/view/disclaimer_view.dart

```diff
@@ -1,206 +1,202 @@
+                            'the time of use 📱.\n\nIt is created using Western Health guidelines, informally '
+                            'peer-reviewed and adapted, with permission, from College/Society guidelines.\n\n',
+                            style: TextStyle(fontSize: 14.0),
+                          ),
+                          Text('🛑 WHAC19 does not constitute official advice\n', style: Styles.textH3),
+                          Text(
+                            'It is your responsibility to ensure your practice is up to date, contextualised for the '
+                            'patient and in accordance with your institution\'s practice 🤓.\n\n',
+                            style: TextStyle(fontSize: 14.0),
+                          ),
+                          Text('Full Disclaimer\n', style: Styles.textH3),
+                          Text(
+                            Strings.disclaimerBody,
+                            //style: Styles.textLegal,
+                            style: TextStyle(fontSize: 14.0),
+                          ),
                         ],
                       ),
-                    )
-                  : Column(
-                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                      crossAxisAlignment: CrossAxisAlignment.stretch,
-                      children: <Widget>[
-                        _content,
-                        if (!snapshot.data) _agreeButton else _agreedText,
-                      ],
                     ),
+                  ),
+                ),
+              ),
             ),
           );
-        });
-  }
-
-  Future<bool> _checkAgreed() async {
-    final prefs = await SharedPreferences.getInstance();
-    return prefs.getBool('disclaimer_agreed') ?? false;
-  }
-
-  // Close the app or close the disclaimer based on whether the user has
-  // agreed to the terms or not
-  bool _closeDisclaimerOrCloseApp(
-      BuildContext context, bool _hasAgreedToTerms) {
-    if (_hasAgreedToTerms) {
-      return _closeDisclaimer(context);
-    } else {
-      // Close App
-      exit(0);
-      return false;
-    }
-  }
-
-  bool _closeDisclaimer(BuildContext context) {
-    if (Navigator.canPop(context)) {
-      // Pop the disclaimer if we are not in the top route
-      return true;
-    } else {
-      // Replace with home if we are in the top route
-      Navigator.pushReplacementNamed(context, Routes.home);
-      return false;
+        },
+      );
     }
-  }
 
-  Future<void> _setAgreed() async {
-    final prefs = await SharedPreferences.getInstance();
-    await prefs.setBool('disclaimer_agreed', true);
-    await prefs.setBool('disclaimer_first_view', true);
+    return SafeArea(
+      bottom: true,
+      top: true,
+      child: Scaffold(
+        appBar: AppBar(
+          backgroundColor: Colors.white,
+          elevation: 4.0,
+          iconTheme: Styles.appBarIconTheme,
+          automaticallyImplyLeading: false,
```

Comment:
You need to implement a check here otherwise the user will not be able to go back if they check the disclaimer from the info page.

Reply:
Yeah thats the bug in the Information screen view. Need to move the FutureBuilder up to get that value in here.

Reply:
I'm still torn on the Disclaimer view at app startup vs Disclaimer view from the Information screen, because the only thing they really have in common is the disclaimer content itself. But happy to keep your original approach of using it for both for now, unless they change more in the future and get even more different 👍

Reply:
All done.